config/finders/all_content.yml

```diff
@@ -8,14 +8,10 @@ rendering_app: finder-frontend
 schema_name: finder
 title: All content
 description: Find content from government
-signup_content_id: 49838f97-5d2e-407a-8876-30c4abe2a6bc
```

Comment:
Shall we also delete the email signup content item from rummager? We should unpublish it, too.

Reply:
Yes, I'll add a point to the trello card mentioning that we should do that as part of deployment